packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -1,6 +1,39 @@
 {
     "version": 1,
-    "timestamp": "2021-07-131T11:20:00+00:00",
-    "sequence": 3,
-    "actions": []
+    "timestamp": "2021-09-16T00:00:00+00:00",
+    "sequence": 4,
+    "actions": [
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "21.9.1",
+                        "mobile": "!",
+                        "web": "!"
+                    }
+                }
+            ],
+            "message": {
+                "id": "40d88610-170f-11ec-9621-0242ac130002",
+                "priority": 91,
+                "dismissible": true,
```

Comment:
`dismissible` means it is gone forever then or it is displayed again on next launch?

Reply:
`dismissible` means that user can click on X to close it, if he does not close, it will be there on next launch

Reply:
But if they click on X it will not be there next time, right? I meant whether it is closed "permanently".

Well anyway let's put it to `false` I think we should push it more. Feel free to force-push then.